apis/addons/v1alpha1/addons_types.go

```diff
@@ -25,16 +25,16 @@ type AddonSpec struct {
 // AddonInstallSpec defines the desired Addon installation type.
 type AddonInstallSpec struct {
 	// Type of installation.
-	// +kubebuilder:validation:Enum={"OwnNamespace","AllNamespaces"}
+	// +kubebuilder:validation:Enum={"olmOwnNamespace","olmAllNamespaces"}
```

Comment:
Does kubebuilder annotation use the variable's name or the actual value? In this case the variable name is `OlmOwnNamespace` but it's value is `olmOwnNamespace`. Maybe make them identical to be safe?

Reply:
Good catch @sugarraysam!
kubebuilder transforms this annotation into the enum for allowed values in the addon CRD's openapi schema [1]

IMO the enum values should be uppercase like before, based on the kubernetes api-conventions doc [2].

[1] https://github.com/erdii/addon-operator/blob/4130031f85456cde05a3fb185171d62d56c02928/config/deploy/addons.managed.openshift.io_addons.yaml#L82
[2] https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#constants

Reply:
This doc also suggests that it should be `OLM` everywhere, and not `Olm` or `olm`. Maybe I should change that too